run.js

```diff
@@ -4,6 +4,27 @@ const util = require('util');
 const package = require('./package.json');
 const { exit } = require('process');
 
+// create file
+fs.open('index.html', 'w', function (err, file) {
+    if (err) throw err;
+    //console.log('Saved!');
+});
+
+
+// function for writing file
+function writeToFile(scr) {
+    fs.writeFile('index.html', scr, function (err) {
+        if (err) throw err;
+        //console.log('Replaced!');
+    });
+}
+
+// variable for html file content
+const htmlDefStart = '<!DOCTYPE html><html><body>'
+const htmlDefEnd = '</body></html>'
+const htmlTableStart = '<table><tr><th>Testcase #</th><th>Input</th><th>Result</th><th>Expected</th><th>Actual</th></tr>'
+var cont = ''
+
```

Comment:
Let's not have global variable, and have them at the start of the function where the HTML is actually being generated.